src/Parallel/ParallelComponentHelpers.hpp

```diff
@@ -141,6 +142,17 @@ using get_option_tags = tmpl::remove_duplicates<tmpl::flatten<tmpl::transform<
     InitializationTagsList,
     detail::get_option_tags_from_initialization_tag<tmpl::_1>>>>;
 
+/// \ingroup ParallelGroup
+/// \brief Given a list of simple tags and a tagged tuple containing items
+/// created from input options, return a tagged tuple of items constructed
+/// by calls to create_from_options
+template <typename... SimpleTags, typename... OptionTags>
+tuples::TaggedTuple<SimpleTags...> get_from_options(
+    tmpl::list<SimpleTags...> /*meta*/,
+    const tuples::TaggedTuple<OptionTags...>& options) noexcept {
+  return {SimpleTags::create_from_options(options)...};
```

Comment:
Why not unpack the specific options for each simple tag at this level rather than passing in a tagged tuple? That would prevent people from accidentally using an option in a tag that's not listed in `option_tags`. What I mean is (roughly)
```cpp
{SimpleTags::create_from_options(get<SimpleTags::option_tags>(options)...)...};
```
where I'm (incorrectly) unpacking the list `SimpleTags::option_tags`. You'll need an `impl` function to handle this, but I think it's safer than the current implementation since it eliminates another class of bugs.